ontology/yaml/resources/LIGHTING/entity_types/ABSTRACTS.yaml

```diff
@@ -0,0 +1,722 @@
+GLC1:
+  id: "828b00f4-da21-457f-8265-612d805ff877"
+  description: "Group lighting control for gateways using module 1 on the dali device. Each module contains up to 16 groups and 64 device addresses, which consist of light fittings and sensors"
+  is_abstract: true
+  opt_uses:
+  - group_dimmer_percentage_command_1_1
```

Comment:
This is not an abstract type, since it doesnt represent a common functionality or unit atomic grouping of points... there aren't even any required fields on these. These need to be removed.